autoexec.bat.command_com.template

```diff
@@ -0,0 +1,24 @@
+@ECHO OFF
+PROMPT $p$g
+cd c:\flash
+cls
+echo.
+echo.
+echo This will flash your EC firmware to support a classic 7-row keyboard (default)
+echo or other modifications as configured at modification time.
+echo.
+echo.
+echo WARNING: Incorrectly flashing your firmware can damage your laptop.
+echo Ensure this is the right firmware version before attempting to flash
+echo the firmware - there are minimal checks done.
+echo.
+echo This disc will flash the file __DIR\__FL2
+echo.
+echo This disk is prepared for __DESC
+echo.
+echo Buildinfo: __BUILDINFO
+echo Built: __BUILT
+echo.
+pause
+echo on
+command.com

```

Comment:
Looking closer, it seems that /FLASH/COMMAND.COM is a compiled script which calls /FLASH/UPDTFLSH.EXE - another compiled script - which in turn calls FLASH/PHLASH16.EXE - and it looks like that one has some commandline options for specifying which files to flash.

I also wonder if the default flash scripts will allow you to re-flash a "same version" image - the xx30 tools certainly didnt, which is why zmatt worked out the dosflash commandline options.

Either way, to clean this up, it looks like some thought will be needed on how to make the autoexec bat generation better in the future.